tower_eval/cli.py

```diff
@@ -169,22 +177,25 @@ def run_evaluations(configs: dict) -> dict:
                         model_type,
```

Comment:
let's fix the part of get languages in the `get_eval_args_given_task`. Currently, we are doing it with the old string splitting with "-", which will not work for the language variants, like `es-latam` or `pt-br`.